src/instruments/src/EFB/index.jsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * A32NX
+ * Copyright (C) 2020-2021 FlyByWire Simulations and its contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import ReactDOM from 'react-dom';
+import { useState } from 'react';
+import Efb from './Efb.tsx';
+import { renderTarget, useUpdate, getSimVar } from '../util.mjs';
+import logo from './Assets/fbw-logo.svg';
+
+import './Assets/Boot.scss';
+
+// TODO: Move anything dependent on ac power change to A32NX_Core
+function powerAvailable() {
```

Comment:
Just a silly question... Why does the iPad-like device care about the aircraft's power being available? Does it not have its own little battery?

Reply:
It will have that logic later, the aircraft power thing is temporary